Automattic/sensei
# Update Importer REST API with `{job_id}` prefix to most endpoints

### Changes proposed in this Pull Request

* This is definitely open to discussion, but as I was working on the log endpoint, it started bugging my more that `{job_id}` wasn't a URL parameter. I think when I initially did this, I thought the endpoints would only handle the active job, but when it is complete we need to be able to fetch a recently complete job.
* This shifts to these endpoints:
  * `POST /sensei-internal/v1/import` (unchanged; unused endpoint for now, but could be used to create job early)
  * `GET /sensei-internal/v1/import/{job_id|active}` (Allows active because the first call won't know current job_id)
  * `GET /sensei-internal/v1/import/{job_id}`
  * `POST /sensei-internal/v1/import/{job_id|active}/file/{file_level_key}` (Allows active because the first call should create the job. We are doing this now instead of an initial `POST /sensei-internal/v1/import` so we don't clutter things up with unused jobs).
  * `DELETE /sensei-internal/v1/import/{job_id}/file/{file_level_key}`
  * `POST /sensei-internal/v1/import/{job_id}/start`
* Fixes issue where media helper functions aren't loaded when using WP Cron instead of Action Scheduler.

### Testing instructions

* Observe tests passing.
* Observe no regression in frontend import.
